assets/schemas/person/latest

```diff
@@ -1 +1 @@
-v202108.01/
\ No newline at end of file
+v202108.01/
```

Comment:
.

Reply:
I haven't changed anything here. This file already existed before.